tests.c

```diff
@@ -11901,6 +11901,53 @@ test_decode_response_header_kmip_2_0(TestTracker *tracker)
     return(result);
 }
 
+int
+test_compare_query_functions(TestTracker *tracker)
+{
+    TRACK_TEST(tracker);
+
+    uint8 encoding[32] = {
+        0x42, 0x00, 0x74, 0x05, 0x00, 0x00, 0x00, 0x04,
+        0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00,
+        0x42, 0x00, 0x74, 0x05, 0x00, 0x00, 0x00, 0x04,
+        0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x00
+    };
```

Comment:
You don't necessarily need to use an encoding/decoding scheme to test query function comparisons. You can just build the lists manually yourself and compare that way.

Reply:
I changed the test to use two lists.